addons/point_of_sale/models/ir_sequence.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+# Copyright (C) 2004-2008 PC Solutions (<http://pcsol.be>). All Rights Reserved
+from odoo import models, api, _
+from odoo.exceptions import UserError
+
+
+class IrSequence(models.Model):
+    _inherit = 'ir.sequence'
+
+    @api.multi
+    def write(self, vals):
+        for sequence in self:
+            if 'active' in vals and self.env['pos.session'].search_count([('config_id.sequence_id', '=', sequence.id), ('state', '!=', 'closed')]):
```

Comment:
`if 'active' in vals` or `if not vals.get('active', True)` ?

Reply:
It may indeed be better to block only archiving, so if they manage to archive one way or another they can activate it again. good point :+1: